requirements.yaml

```diff
@@ -0,0 +1,7 @@
+dependencies:
+  - name: mysql
+    repository: https://kubernetes-charts.storage.googleapis.com
+    version: 1.2.0
+    import-values:
```

Comment:
Why is this needed?

Reply:
This is the way to tell helm that this chart depends on mysql, and to deploy it before the chart

Reply:
Adding it as a dependency will automatically deploy it before the chart. Import-Values doesn't help that.

Reply:
Removed that in the last-commit